formats/organisations_homepage.jsonnet

```diff
@@ -0,0 +1,18 @@
+(import "shared/default_format.jsonnet") + {
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {},
+    },
+  },
+  links: (import "shared/base_links.jsonnet") + {
```

Comment:
Not a blocker, but you could make these edition links so that they go into the publishing-api in a single call.

Reply:
I decided to not make them edition links since there will be a process that will update the links when new organisations are created etc which can just ran `patch_links` without having to republish the entire content item since that will very rarely change.